homeassistant/components/meraki/device_tracker.py

```diff
@@ -123,10 +125,15 @@ def _handle(self, hass, data):
                 attrs["seenTime"] = i["seenTime"]
             if i.get("ssid", False):
                 attrs["ssid"] = i["ssid"]
+   
+                
+            if i.get("rssi", False):
+                attrs["rssi"] = i["rssi"]
```

Comment:
We shouldn't add RSSI as a state attribute. This measurement is known for spamming the state machine.